app/dao/inbound_sms_dao.py

```diff
@@ -62,6 +63,35 @@ def dao_count_inbound_sms_for_service(service_id, limit_days):
     ).count()
 
 
+def _insert_update_notification_history(subquery, query_limit=10000):
```

Comment:
Will look at this function after lunch

Reply:
Cool, if it makes it easier, I based that function on an existing function doing similar thing for Notifications: https://github.com/alphagov/notifications-api/blob/master/app/dao/notifications_dao.py#L373

Reply:
Thanks, that was useful to see